sirepo/pkcli/roles.py

```diff
@@ -64,7 +81,7 @@ def list_roles(*args):
 # TODO(e-carlin): This only works for email auth or using a uid
 # doesn't work for other auth methods
 @contextlib.contextmanager
-def _parse_args(uid_or_email, roles):
+def _parse_args(uid_or_email, roles=[]):
```

Comment:
This is a Python anachronism. `roles` is statically bound to a specific `[]` instance. Since `[]` is mutable, it is dangerous to do that. You could use `None` or `()` here.

Reply:
[609c03c](https://github.com/radiasoft/sirepo/pull/7070/commits/609c03c32717dcd27067fe26bc7c2bbb4ee16dc1)